utils/build-toolchain

```diff
@@ -97,6 +97,7 @@ YEAR=$(date +"%Y")
 MONTH=$(date +"%m")
 DAY=$(date +"%d")
 TOOLCHAIN_VERSION="swift-LOCAL-${YEAR}-${MONTH}-${DAY}-a"
+DARWIN_TOOLCHAIN_VERSION="0.0.${YEAR}_${MONTH}_${DAY}"
```

Comment:
Ah, the underscores probably aren't valid in a version number either! Smashing Y-M-D together is probably fine.

Sorry for the runaround on this.

Reply:
Yea, the components need to be a valid `UInt`.

Reply:
So….  DARWIN_TOOLCHAIN_VERSION="0.0.${YEAR}${MONTH}${DAY}”?

Reply:
Yeah, that sounds like our best option. Thanks!